README.md

```diff
@@ -821,633 +462,270 @@ query {
-                .edges { $0
-                    .node { $0
-                        .id()
-                        .src()
-                    }
-                }
-            }
-            .variants(first: 10) { $0
-                .edges { $0
-                    .node { $0
-                        .id()
-                        .price()
-                        .title()
-                        .available()
-                    }
-                }
-            }
-        }
-    }
-}
-
-let task = client.queryGraphWith(query) { response, error in
-    let product  = response?.node as? Storefront.Product
-    let images   = product?.images.edges.map { $0.node }
-    let variants = product?.variants.edges.map { $0.node }
-}
-task.resume()
-```
-
-The corresponding GraphQL query looks like this:
-
-```graphql
-{
-  node(id: "9Qcm9kdWN0LzMzMj") {
-    ... on Product {
-      id
-      title
-      description
-      images(first: 10) {
-        edges {
-          node {
-            id
-            src
-          }
         }
-      }
-      variants(first: 10) {
-        edges {
-          node {
-            id
-            price
-            title
-            available
-          }
-        }
-      }
     }
-  }
 }
 ```
 
-### Checkout [⤴](#table-of-contents)
+For more information on the syntax and capabilities of search queries, see [shopify.dev](https://shopify.dev/docs/api/usage/search-syntax).
 
-After browsing products and collections, a customer might eventually want to purchase something. The Buy SDK does not provide support for a local shopping cart since the requirements can vary between applications. Instead, the implementation is left up to the custom storefront. Nevertheless, when a customer is ready to make a purchase you'll need to create a checkout.
+### Managing Carts
 
-Almost every `mutation` request requires an input object. This is the object that dictates what fields will be mutated for a particular resource. In this case, we'll need to create a `Storefront.CheckoutCreateInput`:
+Once you have your products displayed, you need a way for your prospective buyers to purchase them. This process starts with creating a `Cart`.
 
-```swift
-let input = Storefront.CheckoutCreateInput.create(
-    lineItems: .value([
-        Storefront.CheckoutLineItemInput.create(variantId: GraphQL.ID(rawValue: "mFyaWFu"), quantity: 5),
-        Storefront.CheckoutLineItemInput.create(variantId: GraphQL.ID(rawValue: "8vc2hGl"), quantity: 3),
-    ])
-)
-```
+#### Creating a Cart
 
-The checkout input object accepts other arguments like `email` and `shippingAddress`. In our example we don't have access to that information from the customer until a later time, so we won't include them in this mutation. Given the checkout input, we can execute the `checkoutCreate` mutation:
+To create a `Cart`, you will use the `cartCreate` mutation. If you do not specify any input arguments, you will simply get an empty `Cart`.
```

Comment:
Let's call out that while this is possible, it's not desired. You shouldn't pre-emptively create carts: call cartCreate when you have something to put into the cart.